public/i18n/en.json

```diff
@@ -53,5 +53,18 @@
     "instructions-dialog-message-instruction-external": "Mismatch on external data source: use this option if the error lies in the external data source. When possible, please inform the data source's maintainers of the error or correct the mismatch yourself.",
     "instructions-dialog-message-instruction-both": "Both are wrong: use this option if you find there are errors on both sides. When possible, please use the link to the Wikidata statement to correct the mismatch and/or inform the data source’s maintainers of the error.",
     "instructions-dialog-message-instruction-none": "None of the above: use this option if none of the other options are applicable. This might for example be the case if the mismatch has already been resolved by someone else.",
-    "results-back-button": "Refine Item Selection"
+    "results-back-button": "Refine Item Selection",
+    "faq-button": "More Information",
+    "faq-dialog-title": "More on the Mismatch Finder",
+    "faq-dialog-question-finding-mismatches": "How are mismatches found?",
+    "faq-dialog-answer-finding-mismatches": "The Mismatch Finder shows you the origin of each mismatch in the results table. The mismatches come from a variety of sources, notably:",
+    "faq-dialog-answer-finding-mismatches-sources-1": "Researchers that are analyzing Wikidata’s data;",
+    "faq-dialog-answer-finding-mismatches-sources-2": "Large reusers of Wikidata’s data;",
+    "faq-dialog-answer-finding-mismatches-sources-3": "Invdividual contributors working with Wikidata’s data;",
```

Comment:
I think it's just a copy error. The correct apostrophe for this context is a single quote rather than a backtick, thank you for noticing!

Reply:
Hello! Good catch, it's really just a copy error. It shouldn't be a backtick, rather a single quote (apostrophe). 

As for the suggestion on the full-stop rather than the semicolon, can you elaborate more? To give you some context, these were copy suggestions made by Dan that I left because there was no strong opinion up until now. I would be interested in hearing your opinion :)

Reply:
The word "Individual" has a spelling mistake.